packages/pangea-sdk/pangea/asyncio/request.py

```diff
@@ -30,7 +30,7 @@ async def post(
         endpoint: str,
         result_class: Type[PangeaResponseResult],
         data: Union[str, Dict] = {},
-        files: List[Tuple] = [],
+        files: Optional[List[Tuple]] = [],
```

Comment:
I think I removed `Optional` here in the past due to it was the same working with `None` or the empty list. But now that you added it again I wonder if it make sense to mark it as Optional but with empty list as default value, shouldn't we default to None?

Reply:
Updated to `None` now, it's what we have in the sync version anyways.